01-introduction.Rmd

```diff
@@ -64,17 +64,17 @@ By teaching you its foundations and underlying structure, we aim to empower you
 Early geographers used a variety of tools including rulers, compasses and sextants to advance knowledge about the world. 
 However, until John Harrison invented the marine chronometer in the 18th century it had been impossible to determine the exact longitude at sea ('the longitude problem').
 Prior to his invention ships followed for centuries a line of constant latitude making each journey much longer, more expensive and often also more dangerous.
-Nowadays this seems unimaginable with every smarthphone having a GPS receiver at its disposal.
+Nowadays this seems unimaginable with every smartphone having a GPS receiver at its disposal.
 <!--nowadays part-->
 And there are a multitude of other sensors measuring the world in real-time (satellites, radar, autonomous cars, citizens, etc.).
-For instance, an autonomous car is believed to create one terrabyte of data on a daily base.
+For instance, an autonomous car is believed to create one terabyte of data on a daily base.
```

Comment:
Another typo - should be `daily basis` or more concise and clear still:

```
For instance, an autonomous car could create one terabyte of data per day.
```

If you could add a reference, or just a link, that would be great. Very interesting point, thanks for adding it.